apps/rush-lib/src/examples/Sample_CustomCommandLine.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Add a comment saying something like:

// This example illustrates how to define your own custom command line
// options for Rush.  These options will be passed to the NPM scripts in your
// package.json file.
//
// To use this file, rename it to command-line.json and copy it into to your
// common/config/rush folder.  For more information about custom commands,
// see the documentation at [https://aka.ms/rush](https://aka.ms/rush)